### PR TITLE
Add devRant app

### PR DESCRIPTION
**Pull request by Meadowcottage**
Pull request to add unofficial devRant desktop app to app list.

Official Site: https://www.devrant.io/

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -1298,3 +1298,13 @@
     - "Unity"
     - "Scratch"
   icon: "gausssense.png"
+-
+  name: "devRant.io"
```
Comment (jlord) [inline _data/apps.yml]: I wonder if we should include "(Unofficial)" here and in the description also?

Comment (Meadowcottage) [inline _data/apps.yml]: Updated the description to state that it is an unofficial app.

Comment (jlord) [inline _data/apps.yml]: If you aren't affiliated with `devRant.io` I think you should add it to the `name` on line `1302` as well. Ready to merge after that 👍

File: _data/apps.yml [hunk 2]
```diff
@@ -1298,3 +1298,13 @@
...
+  description: "Unofficial devRant app that let's you connect with fellow developers over fun rants about tech"
```
Comment (50Wliu) [inline _data/apps.yml]: It should be `lets` and not `let's`

Comment (Meadowcottage) [inline _data/apps.yml]: Typo fixed.

File: _data/apps.yml (hunk 1 above)
Comment (Meadowcottage) [inline _data/apps.yml]: Added `(Unofficial)` to name.

**Comment by jlord** [comment]
Thanks!

**Pull request merged**